specifyweb/backend/context/views.py

```
@@ -713,27 +714,53 @@ def _build_system_data(*, filter_not_ready_collections: bool):
-    discipline = collection.discipline if collection is not None else None
-    institution = Institution.objects.get()
+    setup_complete = is_guided_setup_complete()
+
+    spversion = Spversion.objects.first()
+    collection = None
+    discipline = None
+    if setup_complete:
+        try:
+            collection = request.specify_collection
+            discipline = collection.discipline if collection is not None else None
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)